### PR TITLE
[NNCF] Filter pruning

### PR DESCRIPTION
**Pull request by evgeniya-egupova**

### CODE REVIEW

**Review by alexsu52 — COMMENTED** [review]
Cool staff! Please add nncf_quantization_pruning configuration for ATSS model too.

**Comment by nervana-ff** [comment]
Can one of the admins verify this patch?

**Review by druzhkov-paul — APPROVED** [review]

**Comment by druzhkov-paul** [comment]
Just out of curiosity, what happens inside of this runner-hook-hookwrapper logic? Does accuracy aware runner modify the base LR regardless of what original LR updater tries to do?

**Comment by evgeniya-egupova** [comment]
> Just out of curiosity, what happens inside of this runner-hook-hookwrapper logic? Does accuracy aware runner modify the base LR regardless of what original LR updater tries to do?

@druzhkov-paul Yes, every new iteration of accuracy aware pruning increases pruning rate and then fine-tunes the model to decide if accuracy criteria are met. So ,it is important to reset the lr scheduler state in order to increase the base learning rate.

**Comment by alexsu52** [comment]
LGTM
